### DISCUSSION TITLE
Cannot Update Wezterm

### DISCUSSION THREAD

**Discussion by chxp82q** [wezterm/wezterm#573] [Q&A] [answered]
Hello,

With the issue of #566, and after it was solved, I cannot use `wezterm-nightly-bin` on Arch Linux as wezterm won't open to update due to the error that was fixed. I know I can builld from source, but how would I keep using that specific AUR package? 

Thanks!

**Comment by wez** [comment, marked as answer]
If you have another terminal installed, you can use that to install the update.
Alternatively, you can download the AppImage, make it executable using a graphical file manager, and then double click on it to open it

https://wezfurlong.org/wezterm/install/linux.html#installing-on-linux-using-appimage

**Reply by chxp82q** [reply]
Thanks, I found another solution as well - it was to clear the cache of my AUR helper and install the package again, so it would pull from the nightly releases from github.
